Catappult/appcoins-wallet-android
# [task/WAL-65] Billing Address and First Launch analytics

**What does this PR do?**

   Implements analytics for first launch and billing address actions. They are for Rakam only.

**Database changed?**

   No

**How should this be manually tested?**

  Check if the analytics are being sent. The first launch one should only be sent on first launch and not on subsequent launches. The billing address analytics can be tested by using a VPN to US and using a test card (you can ask me if you don't have it).

**What are the relevant tickets?**

  [WAL-65](https://aptoide.atlassian.net/browse/WAL-65)

**Code Review Checklist**
